src/runtime_src/core/pcie/driver/linux/xocl/subdev/xgq.c

```diff
@@ -78,6 +78,16 @@
 static DEFINE_IDR(xocl_xgq_vmr_cid_idr);
 #define XOCL_VMR_INVALID_CID	0xFFFF
 
+/* Retry is set to 200 seconds for SC to be active/ready
+ * On the SC firmware side there is a HW watchdog timer, which will automatically recover the SC
+ * when SC got hung during the bootup.
+ * If SC get hung during bootup, it would take 180 secs to recover and another ~20 secs window
+ * as a buffer time to fetch and get ready with all the sensor data.
+ */
+#define MAX_SC_WAIT_TIMEOUT_SEC     200
+#define SC_WAIT_INTERVAL_MILLI_SEC  1000
```

Comment:
Commonly, the naming convention is like this:
```suggestion
#define SC_WAIT_INTERVAL_MSEC  1000
```
You can use whatever you desire.